src/lekcja6_zadanie3_cena_brutto_produktu/Product.java

```diff
@@ -1,11 +1,26 @@
 package lekcja6_zadanie3_cena_brutto_produktu;
 
-public class Product {
+import java.util.Scanner;
 
+public class Product {
+    Scanner scanner = new Scanner(System.in);
     private String name;
     private String description;
     private double netPrice;
     private String category;
+    private Product[] product;
```

Comment:
To pole powinno zostac usuniete. Produkt nie składa sie z wielu produktów.

Reply:
Poprawione